app/src/main/scala/org/alephium/explorer/api/model/StdInterfaceId.scala

```diff
@@ -64,7 +64,7 @@ object StdInterfaceId {
       case "0002"   => NFTCollection
       case "0003"   => NFT
       case "000201" => NFTCollectionWithRoyalty
-      case "ns"     => NonStandard
+      case ""       => NonStandard
```

Comment:
If one contract's last immutable field is "ALPH", will it be `NonStandard` or `Unknown`?

Reply:
`Unknown`
I didn't know we could have `ALPH`, what is the code? `0000`?